metadata_mapper/lambda_function.py

```diff
@@ -24,14 +28,15 @@ def import_vernacular_reader(mapper_type):
         package="metadata_mapper"
     )
 
-    mapper_type_words = snake_cased_mapper_name.split('_')
+    mapper_type_words = snake_cased_mapper_name.split(".")[-1].split('_')
```

Comment:
Pretty sure the `.split(".")[-1]` is unnecessary here - it's already done on line 24.

Reply:
You're right, copied over from main